docs/services/slam/cartographer/_index.md

```diff
@@ -54,188 +81,380 @@ Follow the instructions below to set up the `cartographer` module on your robot:
+{{% /expand%}}
+
+#### Localize only
+
+In this mode, the cartographer-module on your robot executes the Cartographer algorithm itself locally.
 
-{{% /tab %}}
-{{% tab name="JSON Example" %}}
+The configuration is similar to the configuration for [updating an existing map](#update-existing-map), except instead of setting a `Data capture rate (Hz)` on the camera and movement sensor, you set a `Data polling rate (Hz)` on them, since the `cartographer-module` on your robot will poll the live LiDAR and IMU directly at these rates instead of the data being sent to the cloud.
 
-Go to your robot's page on the [Viam app](https://app.viam.com/).
-Navigate to the **Config** tab on your robot's page and select **Raw JSON** mode.
+{{%expand "Click to view an example JSON configuration for localizing only" %}}
 
-Note: This example includes configuration for both the `rplidar` and `cartographer` modules, since both are required here.
+See [Attributes](#attributes) for details.
 
-```json
+```json {class="line-numbers linkable-line-numbers"}
 {
-  "components": [
+  "modules": [
     {
-      "name": "my-rplidar",
-      "model": "viam:lidar:rplidar",
-      "type": "camera",
-      "namespace": "rdk",
-      "attributes": {},
-      "depends_on": []
+      "type": "registry",
+      "name": "viam_rplidar",
+      "module_id": "viam:rplidar",
+      "version": "0.1.14"
+    },
+    {
+      "type": "registry",
+      "name": "viam_cartographer",
+      "module_id": "viam:cartographer",
+      "version": "0.3.36"
     }
   ],
   "services": [
     {
-      "name": "my-cartographer",
       "type": "slam",
       "namespace": "rdk",
       "model": "viam:slam:cartographer",
       "attributes": {
         "config_params": {
+          "min_range_meters": "0.2",
+          "max_range_meters": "25",
           "mode": "2d"
         },
-        "data_dir": "/Users/my-username/cartographer-directory",
-        "data_rate_msec": 200,
-        "map_rate_sec": 60,
-        "sensors": ["my-rplidar"]
-      }
+        "camera": {
+          "name": "rplidar",
+          "data_frequency_hz": "5"
+        },
+        "enable_mapping": false,
+        "use_cloud_slam": false,
+        "existing_map": "${packages.slam_map.test-map-1}/internalState.pbstream"
+      },
+      "name": "slam"
     }
   ],
-  "modules": [
+  "components": [
     {
-      "type": "registry",
-      "name": "viam_cartographer",
-      "module_id": "viam:cartographer",
-      "version": "0.3.36"
-    },
+      "model": "viam:lidar:rplidar",
+      "type": "camera",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+      "name": "rplidar"
+    }
+  ],
+  "packages": [
     {
-      "type": "registry",
-      "name": "viam_rplidar",
-      "module_id": "viam:rplidar",
-      "version": "0.1.14"
+      "type": "slam_map",
+      "name": "test-map-1",
+      "version": "1697208847",
+      "package": "d1c224e8-483e-4cc7-980f-76b89d8fb507/test-map-1"
     }
   ]
 }
 ```
```

Comment:
Per the paragraph directly above this sample, I would expect a `movement_sensor` object of form `{ "name": <string>, "data_frequency_hz": <int> }` in this configuration too.